.github/workflows/build.yaml

```diff
@@ -218,12 +218,11 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
       - name: Checkout code
         uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - name: Setup golangci-lint
+        uses: golangci/golangci-lint-action@v2
```

Comment:
There is already a cache with golangci-lint-action

Reply:
Is it shared between steps? We run go mod vendor before we do verify code generated by clientgo-gen

Reply:
Also if we reshuffle all these steps one day I'd prefer keeping the cached action config explicit.